src/hotspot/share/utilities/copy.hpp

```diff
@@ -297,6 +298,28 @@ class Copy : AllStatic {
     pd_zero_to_bytes(to, count);
   }
 
+ protected:
+  inline static void _shared_disjoint_words_atomic(const HeapWord* from,
```

Comment:
How about dropping the leading underscore prefix?

Reply:
Yep will do. Was originally intended (similar to other pd code) to indicate this was a private/internal API, but the protected status achieves the same thing. Thanks for looking at it and the help with the disassembly analysis. :)